templates/component/tests.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import {EntityBrowser} from './EntityBrowser'
+import {shallow} from 'enzyme'
+
+const EMPTY_FUNC = () => {}
```

Comment:
To avoid prop validation errors during a test run you often have to pass such empty functions. i added it in the template to have a consistency in the naming and ussage. If you dont need this function you are forced to delete it because eslint does not allow unussed vars.